index.html

```diff
@@ -0,0 +1,69 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet"
+        href="https://fonts.googleapis.com/css2?family=Material+Symbols+Outlined:opsz,wght,FILL,GRAD@20..48,100..700,0..1,-50..200" />
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link
+        href="https://fonts.googleapis.com/css2?family=Inter:wght@300&family=Open+Sans:ital,wght@0,600;1,700&family=Roboto:wght@700&display=swap"
+        rel="stylesheet">
+    <link rel="stylesheet" href="./index.css">
+    <title>Awesome Books</title>
+</head>
+
+<body>
+    <div class="body">
+        <header>
+            <h2 class="header">Awesome Books</h2>
+            <ul class="nav-link">
+                <li class="nav">List</li>
+                <li class="nav">Add new</li>
+                <li class="nav">Contact</li>
+            </ul>
```

Comment:
- [ ] **[OPTIONAL]** When I go through your nav links, the cursor has the **text pointer** type. Wouldn't it be better to change the cursor to **link pointer** for a better user experience?